src/earn/EarnActivePools.tsx

```diff
@@ -52,6 +52,7 @@ export default function EarnActivePools() {
 
   return (
     <Touchable
+      shouldRenderRippleAbove
```

Comment:
for my understanding, we don't seem to use this for the other Touchable, why is this required here?

Reply:
We use it in [`src/points/PointsDiscoverCard.tsx`](https://github.com/valora-inc/wallet/blob/main/src/points/PointsDiscoverCard.tsx#L47) and it seems like the simplest way to get the ripple behavior we want.

Reply:
ok, why is it not needed in DappFeaturedActions, but needed here?

Reply:
Linking some previous discussion around `shouldRenderRippleAbove` for posterity: https://github.com/valora-inc/wallet/pull/5409#discussion_r1604612862.